SimulationAgent/SimulationRunner.cs

```diff
@@ -274,6 +282,28 @@ public void Stop()
             }
         }
 
+        public async Task AddDevice(string deviceId, string modelId)
+        {
+            this.AddCustomDevice(deviceId, modelId);
+        }
+
+        /// <summary>
+        /// Delete a list of devices
+        /// </summary>
+        public async Task DeleteDevices(List<string> ids)
+        {
+            foreach (var device in this.deviceConnectionActors)
+            {
+                var deviceId = device.Value.Client.DeviceId;
```

Comment:
the device Id can be found through the actor:
```
device.Device.Id
```

Reply:
thanks, updated connection actor to use Device.Id. But for telemetry actor I still need to find deviceId